src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -1693,7 +1693,9 @@ public static NameSyntax ParseName(string text, int offset = 0, bool consumeFull
         /// <summary>
         /// Parse a TypeNameSyntax node using the grammar rule for type names.
         /// </summary>
-        public static TypeSyntax ParseTypeName(string text, int offset = 0, bool consumeFullText = true)
+        // Backcompat overload, do not remove
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static TypeSyntax ParseTypeName(string text, int offset, bool consumeFullText)
```

Comment:
why is this not just deferring to the other overload?

Reply:
I wanted to leave existing code exactly as is. It probably _could_ do that though.